Dockerfile

```diff
@@ -0,0 +1,53 @@
+### Note: This DockerFile is adapted from https://github.com/ucb-bar/chipyard/blob/master/.circleci/images/Dockerfile which was adapted from: https://github.com/CircleCI-Public/example-images/openjdk
+# This is a full chipyard setup, which will be built manually on-demand in the Tendermint Hardware Project at https://gitlab.com/virgohardware/core/
+
+FROM ubuntu:18.04
+
+MAINTAINER jacobgadikian@gmail.com
```

Comment:
https://docs.docker.com/engine/reference/builder/#maintainer-deprecated. Additionally, we should point this to a different email. Lets do "Chipyard Developers (https://groups.google.com/forum/#!forum/chipyard)"